### ISSUE TITLE
Default height of collision shapes

### ISSUE THREAD

**Issue by LeXXik** [playcanvas/editor#329]
Most of the default collision shapes match the default height of the corresponding model mesh, except Cone and Cylinder:
![image](https://user-images.githubusercontent.com/5677782/111638125-b9982480-8802-11eb-84ec-577ec5bda2c7.png)

Proposal is to make their default values match the visual mesh shape properties.

**Comment by willeastcott** [comment]
Yeah, I agree with this. I bet @vkalpias can fix this in maybe 5 minutes. 😆

**Comment by vkalpias** [comment]
So the reason for this is that the Capsule has 2m height by default. So either we make the capsule 1m by default in the Engine or we change the Cone and Cylinder to be 2m instead of 1m height. 

The other alternative is to change the Collision height to 1m if you change the type to Cylinder or Cone and only if the current height is 2. That is to make sure that it doesn't change again if you scroll between different types...

**Comment by LeXXik** [comment]
I am not sure I am reading it correctly, but does it mean that the collision height value is being shared among all shapes? I suppose it is because of using a collision component data object to construct the shape and when picking the shape type the height property is picked with whatever value it has in it?

It would be nice to keep Cones and Cylinders of 1m height by default. If the Editor would change the height to 1m when the previous value is 2m, wouldn't that change the user specified 2m to 1m on Cone <-> Cylinder change?

I guess, unless each shape type has an ability to have own defaults, then increasing the default height to 2m is the only option.

**Comment by vkalpias** [comment]
You are correct that there is a single height property shared by all shapes. It might have been better to have a separate height property per shape.

We probably can't change the default height in the engine because that will break existing projects.

So the 2 solutions more appropriate are either to handle it in the editor or introduce new height properties per shape and deprecate the old height property.
